configapp/configapp.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from flask import Flask, render_template, send_from_directory
+
+app = Flask(__name__)
+app.config.from_object('configapp.default_settings')
+
+
+@app.route('/')
+def index():
+    files = []
+    listdir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
@Adman since this call is used more times (i.e. not just here), maybe it could be abstracted above, since its result will not change.